kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -145,9 +145,9 @@ class ResidualBasedBossakDisplacementScheme: public Scheme<TSparseSpace,TDenseSp
     ///@{
 
     /**
-     * Recalculates the Newmark coefficients, taking into account the alpha parameters
-     * @param beta: The Newmark beta coefficient
-     * @param gamma: The Newmark gamma coefficient
+     * @brief Recalculates the Newmark coefficients, taking into account the alpha parameters
```

Comment:
There are two ways to define the alpha 1) n + alpha, 2) n + 1 - alpha. Since the default alpha is currently 0.0, the user needs to dig here to figure out which version it is. Can we make a comment or a more obvious default value so that the user can figure it out more easily?

Reply:
Adding commentary

Reply:
thanks